### DISCUSSION TITLE
What elements can be controlled by user permissions?

### DISCUSSION THREAD

**Discussion by echohiall** [pimcore/pimcore#11122] [Q&A / Support]
Hello everyone,
I have a truble with Academy exam with only 1 questions across the all courses. The reason is that information about it in documentation and videos looks pretty confusing for me. For example, in documentation I found the notice that Perspectives and Custom Views are not intended to be used to restrict access to data. And in the video about Custom layouts there is a notice that Custom layouts is not secure too. I tried 2 weeks to compare this information and to find the answer by myself, but looks like I can't understand a question so I need to ask an advice here. Can you help me, please?
![image](https://user-images.githubusercontent.com/97035938/147939053-39e85863-7700-40f3-93be-146937c6df0e.png)
![image](https://user-images.githubusercontent.com/97035938/147939448-76db962f-4e51-4597-82db-5de1ed302960.png)

**Comment by fashxp** [comment]
Hi,
the question is about what elements can be controlled by permissions ... not what elements can be used for restricting data access. 
And there perspectives also can be activated/deactivated via user permissions.

BR
Christian

**Reply by echohiall** [reply]
Thank you very much, it was not clear for me

**Reply by lamnc-nfq** [reply]
it was not clear to me till now.
What is exactly answer?

**Reply by noolldave** [reply]
@lamnc-nfq  it is already in thread ;) that is custom layouts and perspectives
